modules/auxiliary/scanner/mssql/mssql_ping.rb

```diff
@@ -16,6 +18,12 @@ def initialize
       'License'        => MSF_LICENSE
     )
 
+    register_options(
+      [
+        Opt::RHOST,
+        OptInt.new('THREADS', [true, "The number of concurrent threads (max one per host)", 1]),
+      ])
+    set_sane_defaults
```

Comment:
All these calls to set_sane_defaults would disappear into the mixin that instantiate the client